Dockerfile

```diff
@@ -1,19 +1,20 @@
 # Use the official Docker Hub Ubuntu 14.04 base image
 FROM ubuntu:16.04
 
-# Update the base image
-RUN apt-get update
-RUN apt-get -y upgrade
-RUN apt-get -y dist-upgrade
-
 # Install Timesketch dependencies
-RUN apt-get -y install python-pip python-dev libffi-dev python-psycopg2
+RUN apt-get update \
```

Comment:
The official ubuntu base image is responsible to be upgraded regularly and to contain all the latest security fixes. Moreover `apt-get upgrade` probably won't install some things successfully because of limited root capabilities in a container by default.